### PR TITLE
make broadcasts endpoint visible, remove references to SMS

### PR DESCRIPTION
**Pull request by nicpottier**
Fixes: #114 #111 #99

### CODE REVIEW

**Comment by nicpottier** [comment]
quick review @rowanseymour ?

File: temba/api/views.py [hunk 1]
```diff
@@ -361,10 +365,10 @@ def api(request, format=None):
 
     All pages that return a list of items support filtering by one or more attributes. You define how you want the list
     filtered via request parameters.  Note that when filtering by phone number you will need to use the E164 format
-    and URL encode the + character as %2B. An example to retrieve all the outgoing SMS messages since January 1st, 2013
+    and URL encode the + character as %2B. An example to retrieve all the outgoing messages since January 1st, 2013
     that are in a state of Q or S for the number +250788123123:
 
-        /api/v1/sms.json?after=2013-01-01T00:00:00.000&status=Q,S&direction=O&phone=%2B250788123123
+        /api/v1/messages.json?after=2013-01-01T00:00:00.000&status=Q,S&direction=O&phone=%2B250788123123
```
Comment (rowanseymour) [inline temba/api/views.py]: `phone` is "deprecated" so better to use `urn` in this example

File: temba/api/serializers.py [hunk 2]
```diff
@@ -1042,8 +1042,9 @@ class BroadcastCreateSerializer(serializers.Serializer):
     channel = ChannelField(queryset=Channel.objects.filter(pk=-1), required=False)
 
     def __init__(self, *args, **kwargs):
-        self.user = kwargs.pop('user')
-        self.org = self.user.get_org()
+        if 'user' in kwargs:
```
Comment (rowanseymour) [inline temba/api/serializers.py]: When do we create a `BroadcastCreateSerializer` without a user?

**Comment by rowanseymour** [comment]
Just two notes, but looks fine

File: temba/api/serializers.py (hunk 2 above)
Comment (nicpottier) [inline temba/api/serializers.py]: django-rest-framework does this when you do a GET while accepting text/html to show the documentation.

**Pull request merged**